### PR TITLE
Adding --score to generate batch anomaly scores and using --median in en...

### PR DESCRIPTION
**Pull request by mmerce**
...sembles

### CODE REVIEW

File: bigmler/dispatcher.py [hunk 1]
```diff
@@ -109,6 +109,15 @@ def get_test_dataset(args):
        --test-dataset --test-datasets
 
     """
+    test_dataset_ids = None
+    args.test_dataset_ids = []
+    try:
+        # Parses dataset/id if provided.
+        if args.test_datasets:
+            test_dataset_ids = u.read_datasets(args.test_datasets)
+            args.test_dataset_ids = test_dataset_ids
```
Comment (jaor) [inline bigmler/dispatcher.py]: why do you need a separate `test_dataset_ids` variable here? wouldn't simply:

```python
args.test_dataset_ids = u.read_datasets(args.test_datasets)
```

do the same without an extra variable?

**Comment by jaor** [comment]
looks good: just one little nit

File: bigmler/dispatcher.py (hunk 1 above)
Comment (mmerce) [inline bigmler/dispatcher.py]: yes

Comment (mmerce) [inline bigmler/dispatcher.py]: done

**Pull request merged**